setup.cfg

```diff
@@ -47,7 +47,7 @@ markers=
 filterwarnings=
     error
     ignore::pytest.PytestUnraisableExceptionWarning
-    ignore::RuntimeWarning:desc.compute
+    ignore::RuntimeWarning
```

Comment:
This covers a lot of possible things (eg, missing io attrs), not sure we want to ignore them everywhere

Reply:
this was an accident, did not mean to include in the commit